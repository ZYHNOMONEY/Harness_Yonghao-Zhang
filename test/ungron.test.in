json.name = "Example"
json.version = 1.0
json.features.enabled = true
json.features.items[0] = "item1"
json.features.items[1] = "item2"
json.features.items[2] = "item3"
json.features.details.author = "zyh"
json.features.details.license = "sit"
json.tags[0] = "test"
json.tags[1] = "json"
json.tags[2] = "example"
json.metadata = null