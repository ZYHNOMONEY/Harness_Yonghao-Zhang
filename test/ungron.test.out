{
    "json": {
        "name": "Example",
        "version": 1.0,
        "features": {
            "enabled": true,
            "items": [
                "item1",
                "item2",
                "item3"
            ],
            "details": {
                "author": "zyh",
                "license": "sit"
            }
        },
        "tags": [
            "test",
            "json",
            "example"
        ],
        "metadata": null
    }
}